configurations/create_wusc_dadaab_database.py

```diff
@@ -0,0 +1,114 @@
+from core_data_modules.cleaners import swahili
+
+from src.pipeline_configuration_spec import *
+PIPELINE_CONFIGURATION = PipelineConfiguration(
+    pipeline_name="CREATE-WUSC-DADAAB-POOL",
+    engagement_database=EngagementDatabaseClientConfiguration(
+        credentials_file_url="gs://avf-credentials/avf-engagement-databases-firebase-credentials-file.json",
+        database_path="engagement_databases/WUSC-DADAAB"
```

Comment:
Also this project is called "Pool-WUSC". Should we maintain one repo per pool, combine the pools, or try to configure 2 pools from here?